vendor/modules.txt

```diff
@@ -122,8 +122,8 @@ github.com/google/go-cmp/cmp/internal/diff
 github.com/google/go-cmp/cmp/internal/flags
 github.com/google/go-cmp/cmp/internal/function
 github.com/google/go-cmp/cmp/internal/value
-# github.com/google/go-tpm v0.9.1-0.20230914180155-ee6cbcd136f8
-## explicit; go 1.20
+# github.com/google/go-tpm v0.9.2-0.20240919181259-d96ccf715685
+## explicit; go 1.22
```

Comment:
I am a bit confused about these two lines -- are they commented out? Should they be removed?

Reply:
I can't remember editing modules.txt, will investigate what happened there

Reply:
Afaik this comes from updating the go module and is just part of the package management of vendored dependencies. If you look at the `modules.txt` file you will see this frequently.

Reply:
That's just the format of the `modules.txt` as is being generated by `go mod vendor`. Anything in the vendor folder is autogenerated and should never be manually touched so we should leave it the way it is (plus the updated version numbers in this PR of course)